manifests/backup.pp

```diff
@@ -0,0 +1,34 @@
+# @summary Add a backup job to backup databases via profile_backup.
+#
+# @param backup_cmd 
+#   Command used to backup the database. Output to STDOUT. 
+#   Generally mysqldump.
+#
+# @param backup_cmd_options
+#   Options to add to the command to dump the database
+#   An example would be --single-transaction.
+#
+# @param databases
+#   A list of databases to backup. Default is back up all databases
+#
+# @example 
+#   include profile_mysql_server::backup
+class profile_mysql_server::backup (
+  String            $backup_cmd,
+  String            $backup_cmd_options,
+  Array[String]     $databases,
+) {
+  if empty($databases) {
+    profile_backup::client::add_cmd_job { 'mysql_all_databases':
```

Comment:
In some other NCSA modules we are naming the backup job with the NCSA Puppet profile that is providing the backup job. The idea being it was more clear what Puppet module is providing the backup job.

If that approach makes sense, then I'd suggest changing this:
```suggestion
    profile_backup::client::add_cmd_job { 'profile_mysql_all_databases':
```

And similarly the name in line 28.

Reply:
renamed.